### ISSUE TITLE
workaround for `emptyList component` containers

### ISSUE THREAD

**Issue by AntonPuko** [Brewskey/Brewskey.App#166]
there is react-native bug when empty list Component doesn't take full screen.
https://github.com/facebook/react-native/issues/17944#issuecomment-372279222
I can make workaround for it, instead of using litsts empty component prop, do  something like
```
<view>
  <list />
  {rows.length === 0 && <EmptyComponent />}
</view>
```

**Comment by AntonPuko** [comment]
probably i can't do that way..it fails when screen is srollable :/

**Comment by AntonPuko** [comment]
https://github.com/facebook/react-native/pull/18206#issuecomment-375032807 They fixed it, but it will be released in 0.56 probably.

**Comment by AntonPuko** [comment]
https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md#056 - fixed in that..can upgrade when it goes to stable(july 2 I guess) and I have some time.
